staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -304,6 +298,74 @@ func (w *WebhookAuthorizer) Authorize(ctx context.Context, attr authorizer.Attri
 
 }
 
+func resourceAttributesFrom(attr authorizer.Attributes) *authorizationv1.ResourceAttributes {
+	ret := &authorizationv1.ResourceAttributes{
+		Namespace:   attr.GetNamespace(),
+		Verb:        attr.GetVerb(),
+		Group:       attr.GetAPIGroup(),
+		Version:     attr.GetAPIVersion(),
+		Resource:    attr.GetResource(),
+		Subresource: attr.GetSubresource(),
+		Name:        attr.GetName(),
+	}
+
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.AuthorizeWithSelectors) {
+		// If there's an error while parsing selectors, we can safely ignore the selector
+		// This is because selectors only narrow, so if broader permission is available, the request can be allowed.
+		if selector, selectorErr := attr.ParseFieldSelector(); selectorErr == nil && selector != nil {
```

Comment:
nit: probably shift from `selector != nil` to `len(requirements) > 0` once ParseFieldSelector returns requirements

Reply:
updated